apps/src/code-studio/components/SoundLibrary.jsx

```diff
@@ -43,7 +70,9 @@ const styles = {
   },
   categoryArea: {
     float: 'left',
-    marginBottom: 20
+    marginBottom: 20,
+    overflowY: 'scroll',
+    height: 350
```

Comment:
For this change in particular, a screenshot and calling out which browsers / resolutions you tested on would be good.

Reply:
Awesome! I'll go ahead and add screenshot for the scroll bar and height as well as an explanation of why the height was adjusted.

Reply:
- Before the height 350px was added: 
<img width="919" alt="screen shot 2019-03-04 at 12 56 14 pm" src="https://user-images.githubusercontent.com/17921445/53762431-f774bb00-3e7c-11e9-9b6b-d725efb51c58.png">
- A screenshot of how it looks after the height was applied is shown in the very first comment at the top of this page. 350 px cuts the categories in half and let's the user only have to scroll once to look at the other half of the available categories.
- Scroll bar was added to match the style of other dialogs.

Reply:
:+1: Awesome, that makes sense.